base/record_replay.h

```diff
@@ -97,7 +98,7 @@ void OnNavigationEvent(const char* kind, const char* url);
 int NewIdMainThread(const char* name);
 int NewIdAnyThread(const char* name);
 
-
+bool AllowCrossDomainAccesses();
```

Comment:
If we're making this part of the recordreplay API let's use the IsInReplayCode() name from RUN-1502, this name is too specific to the bindings code.